airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcState1.java

```diff
@@ -0,0 +1,71 @@
+package io.airbyte.integrations.source.jdbc;
+
+import io.airbyte.integrations.source.jdbc.models.JdbcState;
+import io.airbyte.integrations.source.jdbc.models.JdbcStreamState;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class AbstractJdbcState1 {
```

Comment:
is the `1` suffix expected?

Reply:
hahaha. definitely a wip. it will go away. there was a name collision i was hacking around while developing.